static/app/gettingStartedDocs/react-native/react-native.tsx

```diff
@@ -72,6 +74,27 @@ shopCheckout() {
   }
 }`;
 
+const getReplaySetupSnippet = (params: Params) => `
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  dsn = "${params.dsn.public}",
+  _experiments: {
+    replaysSessionSampleRate: 1.0,
+    replaysOnErrorSampleRate: 1.0,
+  },
```

Comment:
I'm not up to date on the RN sdk but should this still be in `_experiments`?

Reply:
Yes, `_experiments` is correct.